src/herbivoresAndCarnivores.test.js

```diff
@@ -34,8 +34,7 @@ describe('Animal', () => {
 
       expect(carn)
         .toHaveProperty('health', 100);
-    }
-  );
+    });
 
   it(`Herbivore should have 'hide' method which inverts 'hidden' property`,
```

Comment:
[FUNCTION NAMING] - The 'hide' method in the Herbivore class should only set the 'hidden' property to true, not toggle it. The test description for 'hide' method is incorrect as it says it should invert the 'hidden' property. It should be updated to reflect that the 'hide' method sets 'hidden' to true.